File: ios/ConsentScreenConfigOptions.swift
//
//  ConsentScreenConfigOptions.swift
//  ExpoBoxoSdk
//
//  Created by Azamat Kushmanov on 29/10/25.
//

import ExpoModulesCore

struct ConsentScreenConfigOptions : Record {
    @Field
    var title: String?

    @Field
    var noRequiredFieldsDescription: String?

    @Field
    var requiredFieldsDescription: String?

    @Field
    var allowButtonTitle: String?

    @Field
    var cancelButtonTitle: String?
}

File: android/src/main/java/io/boxo/expo/ConsentScreenConfigOptions.kt
package io.boxo.expo

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

class ConsentScreenConfigOptions : Record {
    @Field
    val title: String? = ""

    @Field
    val noRequiredFieldsDescription: String? = ""

    @Field
    val requiredFieldsDescription: String? = ""

    @Field
    val allowButtonTitle: String? = ""

    @Field
    val cancelButtonTitle: String? = ""
}
